src/local-introspection.js

```diff
@@ -1,67 +1,72 @@
 const debug = require('debug')('token-introspection');
+const JwksClient = require('jwks-rsa');
 const jwk2pem = require('pem-jwk').jwk2pem;
 const jwt = require('jsonwebtoken');
 const promisify = require('util.promisify');
 
 const jwtVerify = promisify(jwt.verify);
 
-function findCandidateKeys(jwtHeader, keys) {
-  function alg2keyType(alg) {
-    if (alg.startsWith('HS')) {
-      return 'oct';
-    } else if (alg.startsWith('RS') || alg.startsWith('PS')) {
-      return 'RSA';
-    } else if (alg.startsWith('ES')) {
-      return 'EC';
-    }
-    return null;
+module.exports = (options) => {
+  let jwksFetchKey = null;
+  if (options.jwks) {
+    debug('Configured JWKS with static keys');
+    jwksFetchKey = (key) => {
+      const k = options.jwks.keys.find(obj => obj.kid === key);
+      if (k) {
+        return Promise.resolve(Object.assign({ rsaPublicKey: jwk2pem(k) }, k));
```

Comment:
* Minor optimisation, but now that we have the chance:
Could all keys be transformed beforehand since these static keys never change during runtime?
So do `jwk2pem`, store in an Object by key id, then do the key lookup in that object instead of having to 
use `find` on the entire array of keys.
* Also `node-jwks-rsa` [only includes the key id](https://github.com/auth0/node-jwks-rsa/blob/a81f0bff553519199a8eac4fe78492da52a4e4a1/src/JwksClient.js#L55-L57) together with the PEM data (not the entire JWK as here with `k`).
* This doesn't support static JWK with X509 cert chain in `x5c` (which it definitely didn't do before either, but now that we support it for keys fetched from jwks_uri it would be a nice to have... Maybe...).

(Record number of comments on a single line 😛  ?)